docs/platformdev_blackpill_f411.md

```diff
@@ -1,4 +1,4 @@
-# WeAct Blackpill (STM32F411)
+# WeAct Blackpill (STM32F411 & STM32F401)
```

Comment:
```suggestion
# WeAct Blackpill (STM32F4x1)
```

Reply:
Been there, done that. Added a sentence including F401 and F411 for searchers.